test/integration/concierge_impersonation_proxy_test.go

```diff
@@ -1847,6 +1848,76 @@ func TestImpersonationProxy(t *testing.T) { //nolint:gocyclo // yeah, it's compl
 		}, 2*time.Minute, 500*time.Millisecond)
 	})
 
+	t.Run("using externally provided TLS serving cert with byte arrays", func(t *testing.T) {
```

Comment:
The previous test is: `using externally provided TLS serving cert`, but this one appens `...with byte arrays`.  Should we have a comment here at least explaining why (for a user) either option works, for our future selves?